pkgs/by-name/op/openjph/package.nix

```
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchFromGitHub, cmake }:
+
+stdenv.mkDerivation rec {
+  pname = "openjph";
+  version = "0.14.2";
+
+  src = fetchFromGitHub {
+    owner = "aous72";
+    repo = pname;
+    rev = version;
+    hash = "sha256-jksQfNcgw3LRzQlcsGpTAn6avmM3BsH6wGNTdX9khYY=";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  outputs = [ "out" "dev" ];
+
+  cmakeFlags = [
+    "-DOJPH_ENABLE_TIFF_SUPPORT=OFF"
+  ];
+
+  strictDeps = true;
+
+  meta = with lib; {
```

```suggestion
  meta = {
```
https://github.com/NixOS/nixpkgs/issues/208242